_books/ion-1-1/src/modules/system_module.md

```diff
@@ -10,8 +10,20 @@ This chapter catalogs the system-provided symbols and macros.
 The examples below use unqualified names, which works assuming no other macros with the same name are in scope. The unambiguous form `$ion::macro-name` is always available to use in the [template definition language](../macros/macros_by_example.md).
 <!-- TODO: replace the above link with a TDL-specific reference once we have one. /-->
 
-> [!WARNING]
-> This list is not complete. We expect it to grow and evolve as we gain experience writing macros.
+### Relation to Local Symbol and Macro Tables
+
+In Ion 1.0, the system symbol table is always the first import of the local symbol table.
+However, in Ion 1.1, the system symbol and macro tables have a system address space that is distinct from the local address space.
+When starting an Ion 1.1 segment (i.e. immediately after encountering an `$ion_1_1` version marker), the local symbol
+table is prepopulated with the system symbols. 
+(Why? They can also be used via opcode `EE` which requires the same number of bytes as the `E1` OpCode.) 
+The local macro table is also prepopulated with the system macros.
+(This time it makes sense because we're actually saving a byte.)
+However, the system symbols and macros are not permanent fixtures of the local symbol and macro tables respectively.
+
+
+When a local macro has the same name as a system macro, it shadows the system macro.
+It is not possible to invoke the system macro unless that system macro has been given an alias in the local macro table.
```

Comment:
This line is a bit fuzzy to me. In TDL, if `add_symbols` gets shadowed I can still refer to it via `$ion::add_symbols`.  I think this is saying you can't invoke a system macro _via an e-expression_ if it was shadowed in the active macro table. If that's the case you could you make that context explicit?